services/device-registry-base/src/main/java/org/eclipse/hono/deviceregistry/service/device/AbstractDeviceManagementService.java

```diff
@@ -372,6 +406,7 @@ protected String generateDeviceId(final String tenantId) {
                     }
                     return processDeleteDevicesOfTenant(tenantId, span);
                 })
+                // TODO send notification
```

Comment:
are you going to address this as part of this PR or will you create another PR for it?

Reply:
TBH I haven't thought much about this yet and have no idea how to address this. So, yes this should be part of another PR.

Reply:
Ok, then please don't forget to do so ;-) I can imagine using a dedicated notification type for that purpose.